FlowCrypt/Controllers/Setup/SetupManuallyEnterPassPhraseViewController.swift

```diff
@@ -238,8 +238,12 @@ extension SetupManuallyEnterPassPhraseViewController {
     }
 
     private func importKeys(with existedKeys: [KeyDetails], and passPhrase: String) throws {
```

Comment:
`existedKeys` should be called `existingKeys` :-) thanks

Reply:
ok